### PR TITLE
Add possibility to bake modeless form

### PR DESCRIPTION
**Pull request by antograssiot**
and tests for it

### CODE REVIEW

**Comment by lorenzo** [comment]
This could probably just go to master

**Comment by josegonzalez** [comment]
yay

File: src/Shell/Task/TestTask.php [hunk 1]
```diff
@@ -601,7 +603,9 @@ public function getOptionParser()
                 'Component', 'component',
                 'Behavior', 'behavior',
                 'Shell', 'shell',
+                'shell_helper',
```
Comment (markstory) [inline src/Shell/Task/TestTask.php]: Thanks :+1:

**Comment by markstory** [comment]
Looks good :+1:

**Comment by antograssiot** [comment]
I could target master, then I'll have to update 1.1 for shell helper but I'm ok with it if others also think it's worth it

**Comment by markstory** [comment]
@antograssiot It means we can ship form baking faster. I can fix the mistake on `1.1` if its going to be a hassle.

**Comment by antograssiot** [comment]
@markstory no no pb, I'll get this done

**Pull request closed without merging**